app/posts/register-trainee-teachers/2020-09-30-beta-prototype.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Beta prototype v1
+description:
+date: 2020-10-12
+related:
+  items:
+  - text: Current beta prototype (Heroku)
+    href: https://register-prototype.herokuapp.com/
+  - text: Github repo for prototype
+    href: https://github.com/DFE-Digital/register-trainee-teachers-prototype
+  - text: Alpha prototype
+    href: /register-trainee-teachers/alpha-prototype/
+---
+
+## Building a new prototype from scratch
+Starting in early September, we began work on a new prototype for the Register service. The [alpha prototype](/register-trainee-teachers/alpha-prototype/) was a static invision prototype, which while good for quick iteration, limited some of the usability testing we could do. This new one is built with the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/) and allows us to test more complex journeys with more realistic data.
+
+## Journey changes
+Working in html and designing for progressive enhancement necessitated some changes from the alpha designs:
+* use the task list pattern rather than tabs for each form
+* break up forms across pages to support adding multiples (like multiple degrees)
+* implicitly require that each section be completed as a whole, and not allow individual fields to be left blank
```

Comment:
```suggestion
Working in HTML and designing for progressive enhancement necessitated some changes from the alpha designs:

* use the task list pattern rather than tabs for each form
* break up forms across pages to support adding multiples (like multiple degrees)
* implicitly require that each section be completed as a whole, and not allow individual fields to be left blank
```